src/ocean/LibFeatures.d

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+
+    Meta module defining constants that match ocean feature releases allowing
+    downstream libraries to conditionally implement functionality relying on
+    later ocean versions.
+
+    Copyright:
+        Copyright (c) 2009-2016 Sociomantic Labs GmbH.
+        All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE_BOOST.txt for details.
+        Alternatively, this file may be distributed under the terms of the Tango
+        3-Clause BSD License (see LICENSE_BSD.txt for details).
+
+*******************************************************************************/
+
+module ocean.LibFeatures;
+
+const has_features_2_5 = true;
+const has_features_2_4 = true;
```

Comment:
My point is, if somebody has ocean v2.4, then when you check for the constant it won't be there anyway, in that case you'll still be using features from v2.3, not v2.4, right? It will be when people upgrade to v2.7 (or v2.5 if we include it in that one first) that the feature check will work, so why not just checking for ocean v2.7 instead of v2.4 then? If the check will only pass for v2.7 and up?